tests/templates/kuttl/oidc/03-install-test-nifi.yaml.j2

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: test-nifi
```

Comment:
nit(optional): It would be great if we could migrate this to a k8s Job. I have seen to many failing tests because of stuff like this.

Reply:
Awesome, thanks!